### ISSUE TITLE
Enforce field names

### ISSUE THREAD

**Issue by Krobotechnologies** [sethsandaru/vue-form-builder#112]
Please @sethsandaru @phatsandaru  is there a way to enforce field names when building the form? I want to make sure users fill the unique field names before they can create the input field

**Comment by sethsandaru** [comment]
@Krobotechnologies hi mate, you mean it needs to check if the `Unique Field Name` is existing or not before saving?

**Comment by Krobotechnologies** [comment]
Yes please, I know if the user doesn't fill the unique name, it will use an id for the field but I want the user to always fill the unique name, so in my case, if the name is not filled then you create the field and an error message will be flashback to the user

**Comment by sethsandaru** [comment]
@Krobotechnologies of course, thanks for the suggestion
